accessor/accessor_helper.hpp

```diff
@@ -30,23 +30,23 @@ THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT
 OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 ******************************<GINKGO LICENSE>*******************************/
 
-#ifndef GKO_CORE_BASE_ACCESSOR_HELPER_HPP_
-#define GKO_CORE_BASE_ACCESSOR_HELPER_HPP_
+#ifndef GKO_ACCESSOR_ACCESSOR_HELPER_HPP_
+#define GKO_ACCESSOR_ACCESSOR_HELPER_HPP_
 
 
 #include <array>
+#include <memory>
 #include <tuple>
 #include <type_traits>
 #include <utility>
 
 
-#include <ginkgo/core/base/dim.hpp>
-#include <ginkgo/core/base/range.hpp>
-#include <ginkgo/core/base/types.hpp>
+#include "accessor/index_span.hpp"
+#include "accessor/utils.hpp"
```

Comment:
Maybe something to consider: Make this relative to allow easy relocation
```suggestion
#include "index_span.hpp"
#include "utils.hpp"
```